### PR TITLE
Improve A11y

### PR DESCRIPTION
**Pull request by finnp**
Hey,

I fixed a few accessibility issues on berlin.js, including low contrast and un-structured heading usage.

Best,
Finn

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -168,6 +175,7 @@ <h3 class="ug-info triangle-border left">
           </p>
         </div>
       </div>
+      <
```
Comment (rmehner) [inline index.html]: Is that a leftover `<`? :)

Comment (finnp) [inline index.html]: That's the horn of the devil :imp:  - Just updated it.

**Comment by rmehner** [comment]
Thank you very much :)
1. I'd love to have a better visual difference between a free slot and a taken slot. This is what we have now / before your PR:

berlinjs_ _berlin_s_finest_javascript_usergroup
1. The logo is not loading when you open up the index.html locally in your browser. One of the main advantages is that everyone who opens up a PR can very easily test the website locally.

Thank you again for all your work, definitely looking forward to merging this :)

**Comment by finnp** [comment]
Thanks for reviewing. Those changes were accidental, now it looks like this:

berlinjs_ _berlin_s_finest_javascript_usergroup

**Comment by rmehner** [comment]
Thank you again :)

**Pull request merged**